_html/scss/base/_elements.scss

```diff
@@ -114,6 +114,18 @@ button,
             padding: 1.25rem 5rem;
         }
     }
+
+    &--default {
+        padding: 0;
+        background-color: transparent;
+        color: $primary-color;
+        font-size: $global-font-size;
+
+        @include on-events {
+            background-color: transparent;
+            color: $primary-color;
```

Comment:
Shouldn't state change on hover?

Reply:
Not really, i just wanted to have defaults a tags with different colors for the sponsors which have default set colors. (gold, platinum etc.)